### PR TITLE
Quick galaxy fixes

### PR DESCRIPTION
**Pull request by dpryan79**
Fix tmp_dir and extendReads.

### CODE REVIEW

**Comment by dpryan79** [comment]
@bgruening: This seems to work now. Can you have a quick look at the changes and see if they seem kosher to you?

File: galaxy/wrapper/deepTools_macros.xml [hunk 1]
```diff
@@ -61,6 +65,25 @@
             </when>
         </conditional>
 
+    <xml name="bigwigCorrelate_mode_actions">
```
Comment (bgruening) [inline galaxy/wrapper/deepTools_macros.xml]: Is this anywhere needed?

File: galaxy/wrapper/deepTools_macros.xml [hunk 2]
```diff
@@ -159,14 +182,26 @@
...
+                        Reads are only extended if --extendReads is set to a value greater than the read length. *NOTE*: For spliced-read data, this option is not
+                        recommended as it will extend reads over skipped regions, e.g. introns in RNA-seq data.">
+                <option value="no" selected="True">No extension. The default value and most typically appropriate.</option>
+                <option value="yes">Paired-end extension. Suitable only for paired-end datasets.</option>
+                <option value="custom">A custom length, which will be applied to ALL reads.</option>
+            </param>
+            <when value="no" />
+            <when value="yes" />
+            <when value="custom">
+                <param name="extendReadsValue" type="integer" value="300" min="1"
```
Comment (bgruening) [inline galaxy/wrapper/deepTools_macros.xml]: Can you please change `name=` to `argument=`

**Comment by bgruening** [comment]
@dpryan79 I fixed my own comments and removed one macro. I think this is not needed, please put it back if I'm misleading. I will update now everything before I jump into the train. So a new Docker Image is triggered.

**Comment by dpryan79** [comment]
@bgruening You're correct that it didn't seem needed. Thanks

**Pull request merged**